src/arrow2/src/io/parquet/read/schema/mod.rs

```diff
@@ -42,13 +51,13 @@ impl Default for SchemaInferenceOptions {
 /// This function errors iff the key `"ARROW:schema"` exists but is not correctly encoded,
 /// indicating that that the file's arrow metadata was incorrectly written.
 pub fn infer_schema(file_metadata: &FileMetaData) -> Result<Schema> {
-    infer_schema_with_options(file_metadata, &None)
+    infer_schema_with_options(file_metadata, None)
 }
 
 /// Like [`infer_schema`] but with configurable options which affects the behavior of inference
 pub fn infer_schema_with_options(
     file_metadata: &FileMetaData,
-    options: &Option<SchemaInferenceOptions>,
+    options: Option<SchemaInferenceOptions>,
```

Comment:
Why do we need to own?

Reply:
It's considered un-idiomatic in Rust to pass in an `&Option<T>`. Rather, it's preferred to use `Option<&T>` or `Option<T>` (assuming the type is owned or copy-able).

I can change it to `Option<&T>` instead of moving into each function call instead.

Reply:
Sounds good